src/component/admin/auth/simple-auth.tsx

```diff
@@ -0,0 +1,96 @@
+import React, { useState, useContext, useEffect } from 'react';
+import {
+    Button,
+    FormControlLabel,
+    Grid,
+    Switch,
+} from '@material-ui/core';
+import { Alert } from '@material-ui/lab';
+import PageContent from '../../common/PageContent/PageContent';
+import AccessContext from '../../../contexts/AccessContext';
+import { ADMIN } from '../../providers/AccessProvider/permissions';
+import useAuthSettings from '../../../hooks/api/getters/useAuthSettings/useAuthSettings';
+import useAuthSettingsApi, {ISimpleAuthSettings } from '../../../hooks/api/actions/useAuthSettingsApi/useAuthSettingsApi';
+
+function SimpleAuth() {
```

Comment:
It will be good if we conserve the same pattern of how we create our components. Which are all arrow function components.

Reply:
Good point. I would also prefer if we kept the naming convention of components `simple-auth.tsx -> SimpleAuth.tsx`.

Reply:
`PasswordAuthSettings.tsx` is probably more accurate!